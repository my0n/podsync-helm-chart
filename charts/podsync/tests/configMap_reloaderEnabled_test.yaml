suite: test configMap
templates:
  - configMap.yaml
tests:
  - it: should match snapshot
    set:
      configuration.reloaderEnabled: true
    asserts:
      - matchSnapshot: {}
