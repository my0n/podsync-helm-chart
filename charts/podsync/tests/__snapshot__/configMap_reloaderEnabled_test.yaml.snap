should match snapshot:
  1: |
    apiVersion: v1
    data:
      config.toml: |
        # template
    kind: ConfigMap
    metadata:
      annotations:
        reloader.stakater.com/match: "true"
      name: RELEASE-NAME-podsync
